functions-and-scope.js

```diff
@@ -22,26 +30,70 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Zorg ervoor dat jouw functie ook werkt als we een andere array met eindcijfers willen checken, zoals bijvoorbeeld: [6, 4, 5] of [8, 9, 4, 6, 10].
 // Log het antwoord in de terminal.
 
+function cumLaude(gradesArray) {
+    let count = 0;
+    for (let i = 0; i < gradesArray.length; i++) {
+        if (gradesArray[i] >= 8) {
+            count++;
+        }
+    }
+    return count;
+}
+
+const timesCumLaudeOne = cumLaude(grades);
+const timesCumLaudeTwo = cumLaude([6, 4, 5, 6, 6, 6, 6, 6,]);
+const timesCumLaudeThree = cumLaude([8, 9, 4, 6, 10]);
+
+console.log(timesCumLaudeOne);
+console.log(timesCumLaudeTwo);
+console.log(timesCumLaudeThree);
+
 // ---- Verwachte uitkomsten:
 // cumLaude(grades) geeft 6
 // cumLaude([6, 4, 5]) geeft 0
 // cumLaude([8, 9, 4, 6, 10]) geeft 3
 
 
-
-
 /* Opdracht  2: Gemiddeld cijfer */
 
 /* 2a: Script schrijven  */
 // De studenten-administratie moet ieder blok opnieuw berekenen wat het gemiddelde eindcijfer is, maar we beginnen met de grades array van hierboven.
 // Schrijf de stapjes om dit te kunnen berekenen eerst uit en vraag jezelf de volgende dingen af:
-// * Hoe wordt een gemiddelde berekend?
-// * Wat moet ik verzamelen uit de array van cijfers om uiteindelijk een gemiddelde te kunnen berekenen?
+
+
+// * Hoe wordt een gemiddelde berekend? -->  sumOf grades delen door (last index + 1)
+// * Wat moet ik verzamelen uit de array van cijfers om uiteindelijk een gemiddelde te kunnen berekenen? Totaal en dat delen door index + 1
 // * Hoe zorgt ik ervoor dat ik alle waardes uit de array kan langslopen, ook als de array wel 100 entries zou bevatten?
 // Log het antwoord in de terminal.
-
 // ---- Verwachte uitkomst: 6.642857142857143
 
+let total = 0;
+
+for (let i = 0; i < grades.length; i++) {
+    total = total + grades[i];
```

Comment:
Dit kun je nog iets versimpelen
```suggestion
    total += grades[i];
```